CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 6.0.1 (2021-7-20)
```

Comment:
FYI, we either avoid dates in release notes or don't go more granular than a month - since they're easy to get wrong, can change before release, and git history is the real source of truth.

Reply:
I'll see how much trouble it ends up being.  I do like having a detailed release record that can be used outside of GitHub.